packages/control-input/src/ControlGroup.tsx

```diff
@@ -20,6 +20,8 @@ export type ControlGroupProps = PropsWithChildren<{
 	invalid?: boolean;
 	/** Describes the purpose of the field. */
 	label?: string;
+	/** If true, the secondary label will not be rendered.  */
```

Comment:
A suggestion, as we don't use 'secondary label' as common language throughout the DS

```suggestion
	/** If true, the (optional) label will not be rendered.  */
```

Reply:
Good call, updated 👍